packages/protobuf-test/src/message.test.ts

```diff
@@ -12,23 +12,155 @@
+      packedSint32: [],
+      packedSint64: [],
+      packedFixed32: [],
+      packedFixed64: [],
+      packedSfixed32: [],
+      packedSfixed64: [],
+      packedFloat: [],
+      packedDouble: [],
+      packedBool: [],
+      packedNestedEnum: [],
+      unpackedInt32: [],
+      unpackedInt64: [],
+      unpackedUint32: [],
+      unpackedUint64: [],
+      unpackedSint32: [],
+      unpackedSint64: [],
+      unpackedFixed32: [],
+      unpackedFixed64: [],
+      unpackedSfixed32: [],
+      unpackedSfixed64: [],
+      unpackedFloat: [],
+      unpackedDouble: [],
+      unpackedBool: [],
+      unpackedNestedEnum: [],
+      mapInt32Int32: {},
+      mapInt64Int64: {},
+      mapUint32Uint32: {},
+      mapUint64Uint64: {},
+      mapSint32Sint32: {},
+      mapSint64Sint64: {},
+      mapFixed32Fixed32: {},
+      mapFixed64Fixed64: {},
+      mapSfixed32Sfixed32: {},
+      mapSfixed64Sfixed64: {},
+      mapInt32Float: {},
+      mapInt32Double: {},
+      mapBoolBool: {},
+      mapStringString: {},
+      mapStringBytes: {},
+      mapStringNestedMessage: {},
+      mapStringForeignMessage: {},
+      mapStringNestedEnum: {},
+      mapStringForeignEnum: {},
+      oneofField: { case: undefined },
+      repeatedBoolWrapper: [],
+      repeatedInt32Wrapper: [],
+      repeatedInt64Wrapper: [],
+      repeatedUint32Wrapper: [],
+      repeatedUint64Wrapper: [],
+      repeatedFloatWrapper: [],
+      repeatedDoubleWrapper: [],
+      repeatedStringWrapper: [],
+      repeatedBytesWrapper: [],
+      optionalNullValue: NullValue.NULL_VALUE,
+      repeatedDuration: [],
+      repeatedTimestamp: [],
+      repeatedFieldmask: [],
+      repeatedStruct: [],
+      repeatedAny: [],
+      repeatedValue: [],
+      repeatedListValue: [],
+      fieldname1: 0,
+      fieldName2: 0,
+      FieldName3: 0,
+      fieldName4: 0,
+      field0name5: 0,
+      field0Name6: 0,
+      fieldName7: 0,
+      FieldName8: 0,
+      fieldName9: 0,
+      FieldName10: 0,
+      FIELDNAME11: 0,
+      FIELDName12: 0,
+      FieldName13: 0,
+      FieldName14: 0,
+      fieldName15: 0,
+      fieldName16: 0,
+      fieldName17: 0,
+      FieldName18: 0,
+    };
+
+    msg.optionalTimestamp = Timestamp.now();
+
+    expect(msg).toBeDefined();
+
+    // @ts-expect-error TS2339
+    expect(msg.toBinary).toBeUndefined();
+
+    // Test that PlainMessage is recursive w/r/t to nested fields
     // We want to test that the type system sees this function as undefined even though it's still actually there.  So
     // we expect TS error  TS2339, but add a simple test so Jest doesn't complain there's no expectations.
     // @ts-expect-error TS2339
-    expect(plainTimestamp.toBinary).toBeDefined();
-  });
-  test("removes wkt methods from type system", () => {
-    // Custom methods of well-known types are removed as well.
-    // @ts-expect-error TS2339
-    expect(plainTimestamp.toDate).toBeDefined();
+    expect(msg.optionalTimestamp.toDate).toBeDefined();
```

Comment:
With the change suggested above, we are already proving this, and this line can be removed.